File: java/src/model/chimpos/Gallotronix.java
package model.chimpos;

import model.ataque.Ataque;
import model.ataque.GarraMecanica;
import model.tipo.TipoChinpokomon;

import java.util.ArrayList;
import java.util.Arrays;

public class Gallotronix extends Chinpokomon {

    public Gallotronix(String nombre, Integer nivel, TipoChinpokomon tipo) {
        super(nombre, nivel, 25, new ArrayList<Ataque>(Arrays.asList(new GarraMecanica())), tipo);
    }

}

File: python/model/chipos/Gallotronix.py
from model.ataque.GarraMecanica import GarraMecanica
from model.chipos.Chipokomon import Chipokomon


class Gallotronix(Chipokomon):
    def __init__(self, nombre, nivel, tipo):
        Chipokomon.__init__(self, nombre, nivel, 25, [GarraMecanica()], tipo)
        
    def __str__(self):
        return 'Gallotronix'
